helpers/subscribe.js

```diff
@@ -0,0 +1,26 @@
+import models from '../models';
+
+const { Op } = models.Sequelize;
+const { subscribers } = models;
+
+
+const subscribe = async (userId, subscribeTo) => {
+  const getSubscriber = await subscribers.findOne({
```

Comment:
@fridolinho This is a great helper function. However, since it's an async await one, I think you need to wrap it in "try" and "catch" block, in case an error occurs.

Reply:
Thank you for the feedback, but where i call that function it is wrapped in a try and catch block